NDB.Covid19/NDB.Covid19.iOS/Views/InfectionStatus/InfectionStatusViewController.cs

```diff
@@ -151,7 +151,14 @@ async void UpdateUI()
         async void SetStatusContainerState(bool isRunning)
         {
             UIView statusBar = new UIView(UIApplication.SharedApplication.StatusBarFrame);
-            statusBar.BackgroundColor = isRunning ? ColorHelper.STATUS_ACTIVE : ColorHelper.STATUS_INACTIVE;
+            if (NavigationController.TopViewController is SettingsViewController)
```

Comment:
Hmm...  Mabe invert condition. Check if it is infection status page.. in this form the issue can also occur on other pages.

Reply:
Fixed